Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -109,6 +114,19 @@ protected virtual void OnValidate()
             }
         }
         
+        private void OnAdditiveSceneLoaded(Scene scene, LoadSceneMode loadSceneMode)
+        {
+            if (scene.name == ADDITIVE_USERINTERFACE_SCENE_NAME)
+            {
+                UserInterfaceManager.Initialize();
+            }
+            
+            if (scene.name == ADDITIVE_WEAPON_SCENE_NAME)
```

Comment:
can make this `else if` if you don't want to do a switch statement. Cause `scene.name` cant be both cases (useless check). 

Alternatively, you can do an early return after `UserInterfaceManager.Initialize()`